src/test/CryptographyTest.java

```diff
@@ -0,0 +1,31 @@
+package test;
+
+import com.company.Main;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class CryptographyTest {
```

Comment:
names can be improved. imagine someone reading encrypt_abc_fgh, I think 99% he or she will have questions.

I sometimes use happyPath in names, e.g. encrypt_HappyPath

Reply:
Renamed with using happyPath in test title